extensions/amp-delight-player/0.1/amp-delight-player.js

```diff
@@ -192,11 +197,13 @@ class AmpDelightPlayer extends AMP.BaseElement {
 
   /** @override */
   createPlaceholderCallback() {
-    const placeholder = this.element.ownerDocument.createElement('div');
+    const html = htmlFor(this.element);
+    const placeholder =
+        html`<div placeholder><amp-img layout=fill></amp-img></div>`;
```

Comment:
@aghassemi Even though this makes it technically get the correct placeholder URL, the `src` pattern below appears to be broken :(

Reply:
@xymw has the Url for poster changed?